metadata_service/proxy/atlas_proxy.py

```diff
@@ -688,20 +781,23 @@ def add_badge(self, *, id: str, badge_name: str, category: str = '',
     def delete_tag(self, *, id: str, tag: str, tag_type: str,
                    resource_type: ResourceType = ResourceType.Table) -> None:
         """
-        Delete the assigned classfication/tag from the given table
-        API Ref: /resource_EntityREST.html#resource_EntityREST_deleteClassification_DELETE
-        :param table_uri:
-        :param tag:
-        :return:
+        Removes the Glossary Term assignment from the provided source.
+        :param id: Table URI / Dashboard ID etc.
+        :param tag: Tag Name
+        :return:None
         """
-        try:
-            entity = self._get_table_entity(table_uri=id)
-            guid_entity = self._driver.entity_guid(entity.entity[self.GUID_KEY])
-            guid_entity.classifications(tag).delete()
-        except Exception as ex:
-            # FixMe (Verdan): Too broad exception. Please make it specific
-            LOGGER.exception('For some reason this deletes the classification '
-                             'but also always return exception. {}'.format(str(ex)))
+        entity = self._get_table_entity(table_uri=id)
```

Comment:
this method allows users to remove tags outside AMUNDSEN_USER_TAGS glossary scope. It shouldn't allow that as we have this separation for a reason - to provide both curated and user-managed tags.